collector/time.go

```diff
@@ -0,0 +1,124 @@
+// +build windows
+
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+)
+
+func init() {
+	registerCollector("time", newTimeCollector, "Windows Time Service")
+}
+
+// TimeCollector is a Prometheus collector for Perflib counter metrics
+type TimeCollector struct {
+	ClockFrequencyAdjustmentPPBTotal *prometheus.Desc
+	ComputedTimeOffset               *prometheus.Desc
+	NTPClientTimeSourceCount         *prometheus.Desc
+	NTPRoundtripDelay                *prometheus.Desc
+	NTPServerIncomingRequestsTotal   *prometheus.Desc
+	NTPServerOutgoingResponsesTotal  *prometheus.Desc
+}
+
+func newTimeCollector() (Collector, error) {
+	const subsystem = "time"
+
+	return &TimeCollector{
+		ClockFrequencyAdjustmentPPBTotal: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "clock_frequency_adjustment_ppb_total"),
+			"Total adjustment made to the local system clock frequency by W32Time in Parts Per Billion (PPB) units.",
+			nil,
+			nil,
+		),
+		ComputedTimeOffset: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "computed_time_offset"),
```

Comment:
Having the unit suffix would be nice here
```suggestion
			prometheus.BuildFQName(Namespace, subsystem, "computed_time_offset_seconds"),
```